_docs/releasing/Discontinuing-a-project.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Discontinuing a project
+author: OST
+date: 2020-20-05
+index: 3
+category: Releasing
+---
+
+**Overview of the different options for discontinuing a Zalando open source project, this process is to
+be used for projects which no longer have active maintainers, or projects which Zalando no longer have a motivation
+to maintain, usually due to lack of internal use.**
+
+
+### Summary
+
+- Projects which we have no motivation to maintain, we are obligated to gracefully discontinue
+- There are different options to ensure the project lives on
```

Comment:
What's the value of this point given you list the options in the line below?

Reply:
point removed